legal-api/migrations/versions/ba862cfda9e5_foreign_corp_num_to_foreign_identifier.py

```diff
@@ -0,0 +1,28 @@
+"""foreign_corp_num_to_foreign_identifier
+
+Revision ID: ba862cfda9e5
+Revises: f978e34aa8bb
+Create Date: 2024-01-30 09:50:49.159204
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = 'ba862cfda9e5'
+down_revision = 'f978e34aa8bb'
```

Comment:
This seems to down_revise a revision that doesn't exist, throwing a `KeyError` when trying to run the unit tests. I haven't worked much in the migration scripts but this seems to indicate we are missing one of the migrations?

Reply:
We don't need to bring over any of the new migration scripts most of the time.  

In the legal name branch, we have mostly collapsed all the alembic script updates into a few scripts.

@JazzarKarim you can just leave these out as I believe we have already ported over these changes.  Can you double check that https://github.com/bcgov/lear/blob/feature-legal-name/legal-api/migrations/versions/004f7558063f_second.py in legal name branch have the changes in both of the scripts you cherry picked over for this PR?  If things check out, we can remove these two scripts from the PR.

And thanks for flagging this @leodube-aot !

Reply:
Removed both scripts. 

I confirmed that these changes were already in the legal name branch but in this file: https://github.com/bcgov/lear/blob/feature-legal-name/legal-api/migrations/versions/5238dd8fb805_.py

So, I believe it should be OK to remove them. Thanks for catching that Leo and Argus!